frontend/components/community/options.tsx

```diff
@@ -62,10 +63,10 @@ function OptionCard({ option }: { option: Option }) {
          align="center"
          overflow="hidden"
          minHeight="100%"
-         borderRadius="var(--border-radius-lg)"
+         borderRadius={`${borderRadius.lg}`}
          direction={{ base: 'row', md: 'column' }}
-         boxShadow={{ base: 'var(--shadow-2)', md: 'var(--shadow-3)' }}
-         padding={{ base: 'var(--space-4)', md: '0 0 var(--space-6)' }}
+         boxShadow={{ base: `${shadow[2]}`, md: `${shadow[3]}` }}
+         padding={{ base: `${space[4]}`, md: `0 0 ${space[5]}` }}
```

Comment:
Same question as above - I'm assuming the change from 6 to 5 was intentional, but just wanted to check since the rest mapped.

Reply:
You know what, that one was intentional, but when I went to double check I think it looks better with `space[6]`. Good catch!